packages/airnode-protocol/credentials.example.json

```diff
@@ -62,11 +63,21 @@
       "chainId": 42161,
       "url": "https://arb1.arbitrum.io/rpc"
     },
+    "arbitrum-nova": {
+      "accounts": { "mnemonic": "" },
+      "chainId": 42170,
+      "url": "https://nova.arbitrum.io/rpc"
+    },
     "arbitrum-testnet": {
       "accounts": { "mnemonic": "" },
       "chainId": 421613,
       "url": "https://goerli-rollup.arbitrum.io/rpc"
     },
+    "aurora": {
```

Comment:
What about the testnet

Reply:
Is there a precedence for also deploying to the testnet if the contracts are being deployed the mainnet ?